lib/routemaster/jobs/cache_and_sweep.rb

```diff
@@ -8,7 +8,10 @@ module Jobs
     class CacheAndSweep
       def perform(url)
         Dirty::Map.new.sweep_one(url) do
-          Cache.new.get(url)
+          begin
+            Cache.new.get(url)
+          rescue Errors::ResourceNotFound
```

Comment:
Looks good, but maybe leave a warning? (unless it's too complicated to inject a logger)